src/components/header/header.jsx

```diff
@@ -56,18 +60,18 @@ function Header(props) {
                     </div>
                     <button  className="btn__login" style={{display:user.isLogin ? 'none' : 'block'}} onClick={toggleLogin}>{t('header.login')}</button>
                     <button  className="btn__changelanguage" onClick={handleClickChangeLanguage}>{language}</button>
-                   </div>
+                </div>
             </div>
             <div className="header__bottom">
-                            <ul>
-                                <li>{t('header.nav.movie')}</li>
-                                <li>{t('header.nav.promotion')}</li>
-                                <li>{t('header.nav.news')}</li>
-                                <li>{t('header.nav.intro')}</li>
-                            </ul>
+                <ul>
+                    <li><a href="/movie">{t('header.nav.movie')}</a></li>
```

Comment:
sao vẫn dùng thẻ <a> e. link to đâu mà k dùng

Reply:
Da. a 
Do e để header bên ngoài cái router nên nó k cho dùng thẻ link. để e fix lại